src/components/Inputs/ColorPicker.tsx

```diff
@@ -45,13 +46,7 @@ const ColorPicker = ({ value, onChange }: ColorPickerProps) => {
 
   return (
     <ColorPickerContainer ref={ref}>
-      <Input
-        role="button"
-        tabIndex={0}
-        aria-label={t`Pick a color`}
-        onClick={handlePopupOpen}
-        onKeyPress={handlePopupOpen}
-      >
+      <Input aria-label={t`Pick a color`} onClick={handlePopupOpen}>
```

Comment:
The screenreader (Orca) no longer picks up the label with this change, I'm investigating now

Reply:
Issue is https://github.com/alephium/desktop-wallet/pull/310/files#diff-3ac83cbff04686c428c1043abdb1aa3819c25290ed4aec26be1becc172b0f168R27 does not accept all props (and so `aria-label` is ignored)